js/lib/legacy_functions.js

```
@@ -0,0 +1,156 @@
+};
+
+const makeResultSymbol = (_id, _x, _color, _heightPos, _text, _align) => {
+    const div = makeCssResultSymbol(_id, _x, ``, _heightPos, _text, _align);
+    div.style.color = _color;
+    return div;
+}
+
+const checkArrayVal = (_checkArray, _type, _minLength) => {
+    const checkFlg = hasArrayList(_data, _minLength);
+    if (_type === C_TYP_FLOAT) {
+        if (isNaN(parseFloat(_checkArray[0]))) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)